packages/web3-eth-personal/CHANGELOG.md

```diff
@@ -147,4 +147,10 @@ Documentation:
 
 -   Dependencies updated
 
+## [4.0.9]
+
+### Changed
+
+-   Dependencies updated
```

Comment:
if there are only patch & minor dep updates we can skip changes in these packages,

Reply:
skipped packages that are only patch/minor dep updates